wtss_plugin/controller/files_export/files_export.py

```diff
@@ -74,8 +75,8 @@ def generateCode(self, file_name, attributes):
             for band in attributes.get("bands"):
                 bands_string = bands_string + "'" + str(band) + "', "
             bands_string = bands_string[:len(bands_string)-2] + ")"
-            lat = "{:,.2f}".format(attributes.get("coordinates").get("lat"))
-            lon = "{:,.2f}".format(attributes.get("coordinates").get("long"))
+            lat = str(attributes.get("coordinates").get("lat"))
```

Comment:
You should not cast to string directly. You may face errors related precision cast.
#82